### ISSUE TITLE
Error when using ConvertGeneTable function

### ISSUE THREAD

**Issue by johannesnicolaus** [jernst98/ChromHMM#50]
Hi, I encountered the following error when trying to run `ConvertGeneTable` on genepred file generated using UCSC-tools from GTF file of a custom genome on ChromHMM 1.24.

I got the following error:
```
Exception in thread "main" java.lang.NullPointerException
        at edu.mit.compbio.ChromHMM.ConvertGeneTable.convertGeneTableToAnnotations(ConvertGeneTable.java:216)
        at edu.mit.compbio.ChromHMM.ChromHMM.main(ChromHMM.java:15266)
```

The code I ran:
```
java -mx8000M -jar ChromHMM.jar ConvertGeneTable -nobin -noheader \
        -l chromosome_lengths.tsv \
        -v Bar2_p4.Flye.gm/ANCHORFILES \
        -u Bar2_p4.Flye.gm/COORDS \
        Bar2_p4.Flye.gm.mod.genepred Bar2_p4.Flye.gm Bar2_p4
```

To begin with, I'm not sure how to use custom genome using the ConvertGeneTable function. I'm not sure what to put for the last argument "assembly". 

Any help is greatly appreciated!

**Comment by jernst98** [comment]
Thanks for pointing this out. It looks like there is a bug with the -nobin option added in 1.24. As a workaround until I release a new version you can try adding a first column of dummy values and not use the '-nobin' flag. Assembly should match the assembly identifier you want to use.

**Comment by johannesnicolaus** [comment]
Thanks. I added a column with dummy values and it seems to work now

**Comment by jernst98** [comment]
This is now fixed in Version 1.25.
